bsdfdisk/edit_slice.cgi

```diff
@@ -41,16 +41,19 @@ print &ui_table_row($text{'slice_sstart'},
 print &ui_table_row($text{'slice_send'},
 	$slice->{'startblock'} + $slice->{'blocks'} - 1);
 
+# GPT type selection
 print &ui_table_row($text{'slice_stype'},
 	&ui_select("type", $slice->{'type'},
-		   [ sort { $a->[1] cmp $b->[1] }
-			  map { [ $_, &fdisk::tag_name($_) ] }
-			      &fdisk::list_tags() ]));
+		   [ &list_partition_types() ]));
```

Comment:
@karmantyu, we need to main backward compatibility with older versions of BSD that don't have `gpart` available.